test/specs/mounting-options/scopedSlots.js

```diff
@@ -0,0 +1,40 @@
+import { describeWithShallowAndMount, vueVersion } from '~resources/utils'
+import ComponentWithScopedSlots from '~resources/components/component-with-scoped-slots.vue'
+
+describeWithShallowAndMount('scopedSlots', (mountingMethod) => {
+  if (vueVersion >= 2.5) {
+    it('mounts component scoped slots', () => {
```

Comment:
We have an itIf helper function in resouces/utils. Could you use that instead of if else statement